content/source/docs/cloud/api/admin/settings.html.md

```diff
@@ -86,6 +87,7 @@ Key path                                                                 | Type
 `data.attributes.api-rate-limit`                                         | integer  | 30                        | The number of allowable API requests per second for any client. This value cannot be less than 30. To learn more about API Rate Limiting, refer to the [rate limiting documentation][]
 `data.attributes.send-passing-statuses-for-untriggered-speculative-plans`| bool     | `true`                    | When set to `true`, workspaces automatically send passing commit statuses for any pull requests that don't affect their tracked files.
 `data.attributes.allow-speculative-plans-on-pull-requests-from-forks`    | bool     | `false`                   | When set to `false`, [speculative plans][] are not run on pull requests from forks of a repository. This setting is available in Terraform Enterprise versions v202005-1 or later. It is currently supported for the following VCS providers: GitHub.com, GitHub.com (OAuth), GitHub Enterprise, Bitbucket Cloud, Azure DevOps Server, Azure DevOps Services. To learn more about this setting, refer to the [documentation](/docs/enterprise/admin/general.html#allow-speculative-plans-on-pull-requests-from-forks)
+`data.attributes.default-remote-state-access`    | bool     | `true`                   | When set to `true`, new workspaces will deafult to allowing remote state access to them.
```

Comment:
I'd love to link out to something here that gives a little more information about what this setting actually does! Similar to the way the description for the "allow speculative plans on pull requests from forks" setting links out to https://www.terraform.io/docs/enterprise/admin/general.html#allow-speculative-plans-on-pull-requests-from-forks.

Unfortunately, we haven't written that general description of "what is this admin setting even" yet, so there's nothing to link to 😆 I'd lean towards either adding it to this PR if you're up for it or updating this PR once it's ready.